includes/class-wc-install.php

```diff
@@ -946,6 +946,14 @@ private static function get_schema() {
   slug varchar(200) NOT NULL DEFAULT '',
   PRIMARY KEY  (tax_rate_class_id),
   UNIQUE KEY slug (slug($max_index_length))
+) $collate;
+CREATE TABLE {$wpdb->prefix}wc_reserved_stock (
+	`order_id` bigint(20) NOT NULL,
+	`product_id` bigint(20) NOT NULL,
+	`stock_quantity` double NOT NULL DEFAULT 0,
+	`timestamp` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+	`expires` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
suggest default value change to a future timestamp, like 5 minutes

Reply:
this is now changed to `0000-00-00 00:00:00` to support MySQL 5.6.